Samples~/AvatarCreatorSamples/LegacyAvatarCreator/Scripts/ProfileManager.cs

```diff
@@ -30,10 +31,9 @@ private void OnEnable()
             AuthManager.OnSignedOut += DeleteSession;
         }
 
-
         private void OnDisable()
         {
-            profileUI.SignedOut -= AuthManager.Logout;
+            profileUI.SignedOut += AuthManager.Logout;
```

Comment:
Hmm, shouldn't we remove it, not add it here?

Reply:
yep good catch